mlflow/server/js/src/model-registry/components/ModelVersionTable.tsx

```diff
@@ -226,11 +238,12 @@ export const ModelVersionTable = ({
           }),
           meta: { styles: { flex: 2 }, multiline: true },
           cell: ({ getValue, row: { original } }) => {
+            const mvAliases = aliasesByVersion[original.version] || [];
             return (
               <ModelVersionTableAliasesCell
                 modelName={modelName}
                 version={original.version}
-                aliases={getValue() as string[]}
```

Comment:
The existing logic assumes `SearchModelVersions` will return a list of aliases per model version, which is not guaranteed/unfortunately inconsistent between MLflow backend implementations. In particular, we don't return aliases attached to each model version in `SearchModelVersions` in SQL-based backend stores for performance reasons. It seems we ignore such implications and return them in the FileStore, leading to inconsistency. The fix here is to just rely on the overall list of aliases attached to the parent registered model & reference those here